### DISCUSSION TITLE
Why does DC's Keep Child Care Affordable Tax Credit cost >4x as much in PE vs DC's tax expenditure report?

### DISCUSSION THREAD

**Discussion by MaxGhenis** [PolicyEngine/policyengine-us#2695] [Q&A]
The CPS has known issues, but this is surprising since we're so close on the DC CDCC, which uses similar inputs. I just looked through the logic and it seems to capture the law correctly. Maybe because it's on a separate tax form, take-up is much lower? @martinholmer any ideas?

| Program                            | PolicyEngine Estimate (2023)                                       | DC Tax Expenditure Report (2023)                                    |
|------------------------------------|-------------------------------------------------------------|--------------------------------------------------------------|
| Keep Child Care Affordable Tax Credit | [$3.1M](https://policyengine.org/us/policy?focus=policyOutput.netIncome&reform=19272&region=dc&timePeriod=2023&baseline=2)      | [$0.7M](https://ora-cfo.dc.gov/sites/default/files/dc/sites/ocfo/publication/attachments/2022%20Tax%20Expenditure%20Report.pdf#page=242) |
| Child and Dependent Care Credit        | [$3.7M](https://policyengine.org/us/policy?focus=policyOutput.netIncome&reform=19600&region=dc&timePeriod=2023&baseline=2)      | [$3.5M](https://ora-cfo.dc.gov/sites/default/files/dc/sites/ocfo/publication/attachments/2022%20Tax%20Expenditure%20Report.pdf#page=240) |

NB: The Tax Expenditure Report refers to the Keep Child Care Affordable Tax Credit by its previous name, the Early Learning Child Credit. But the description aligns with the current policy.

**Comment by martinholmer** [comment]
@MaxGhenis asked in issue #2695:
> The CPS has known issues, but this is surprising since we're so close on the DC CDCC, which uses similar inputs.  I just looked through the logic and it seems to capture the law correctly.  Maybe because it's on a separate tax form, take-up is much lower?  Any thoughts?

I agree that the separate form (and the notion by some DC taxpayers that they already got a credit for child care expenses via the DC CDCC) could be holding down the take-up of this credit.  Also, in reality, maybe the care expenses eligible for the DC KCCATC are less than for the CDCC (because they may be limited to only some subset of care providers).  I haven't researched this last possibility, so it is highly speculative.
